src/spec/template_spec.js

```diff
@@ -0,0 +1,279 @@
+    var expectedTitleText = "Información de la Plantilla";
+    var expectedTemplateNameText = "Nombre de la Plantilla";
+    var expectedFromEmailText = "Dirección de Correo del Remitente";
+    var expectedFromNameText = "Nombre del Remitente";
+    var expectedSubjectText = "Asunto";
+    var expectedTemplateEditorTitle = "Edición de la Plantilla";
+    var expectedTemplateEditorDescription = "Aquí podrás crear y editar tu plantilla personalizada.";
+
+    // Act
+    templatePage.clickSpanishFlag();
+    var templateTitleText = templatePage.getTemplatePageTitleText();
+    var templateNameText = templatePage.getTemplateNameLabelText();
+    var templateFromEmailText = templatePage.getTemplateFromEmailLabelText();
+    var templateFromNameText = templatePage.getTemplateFromNameLabelText();
+    var templateSubjectText = templatePage.getTemplateSubjectLabelText();
+    var templateEditorTitleText = templatePage.getTemplateEditorTitleText();
+    var templateEditorDescriptionText = templatePage.getTemplateEditorDescriptionText();
+
+    // Assert
+    expect(templateTitleText).toBe(expectedTitleText);
+    expect(templateNameText).toBe(expectedTemplateNameText);
+    expect(templateFromEmailText).toBe(expectedFromEmailText);
+    expect(templateFromNameText).toBe(expectedFromNameText);
+    expect(templateSubjectText).toBe(expectedSubjectText);
+    expect(templateEditorTitleText).toBe(expectedTemplateEditorTitle);
+    expect(templateEditorDescriptionText).toBe(expectedTemplateEditorDescription);
+  });
+
+  it('should show the save template button in edit mode in english', () => {
+    // Arrange
+    beginAuthenticatedSession();
+    browser.get('/#/templates/32e93fa4-c8f4-467a-a2bb-da95f72a27b4');
+    setupTemplateResponse();
+
+    var templatePage = new TemplatePage();
+
+    // Act
+    var saveTemplateButtonIsDisplayed = templatePage.isSaveButtonDisplayed();
+
+    // Assert
+    expect(saveTemplateButtonIsDisplayed).toBeTruthy();
+  });
+
+  it('should not allow to save template with empty body content', () => {
+    // Arrange
+    beginAuthenticatedSession();
+    browser.get('/#/templates/32e93fa4-c8f4-467a-a2bb-da95f72a27b4');
+    setupTemplateResponse();
+
+    var templatePage = new TemplatePage();
+
+    // Act
+    templatePage.cleanTemplateBodySection();
+    templatePage.clickSaveTemplateButton();
+
+    // Assert
+    expect(templatePage.isInvalidTemplateBody()).toBeTruthy();
+  });
+
+  it('should save template in edit mode and redirect to templates list', () => {
+    // Arrange
+    beginAuthenticatedSession();
+    browser.get('/#/templates/32e93fa4-c8f4-467a-a2bb-da95f72a27b4');
+    setupTemplateResponse();
+    var body = "Test content body";
+
+    var templatePage = new TemplatePage();
+
+    // Act
+    templatePage.FillTemplateBodySection(body);
+    templatePage.clickSaveTemplateButton();
+    
+    // Assert
+    expect(browser.getCurrentUrl()).not.toContain('templates/');
```

Comment:
> _should save template in edit mode and redirect to templates list_
> _expect(browser.getCurrentUrl()).not.toContain('templates/');_

Does not _templates list_ contains _templates/_ in the URL? what is the final URL? why are you not testing for it?